xml/System.Reflection/TypeDelegator.xml

```diff
@@ -1073,9 +1089,15 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <summary>When overridden in a derived class, returns the parameter types of the current function pointer <see cref="T:System.Type" />.</summary>
+        <returns>An array of <see cref="T:System.Type" /> objects representing all the parameter types for the current function pointer <see cref="T:System.Type" />.
+
+ -or-
+
+ An empty array of type <see cref="T:System.Type" />, if no parameters are defined for the current function pointer <see cref="T:System.Type" />.
+        </returns>
         <remarks>To be added.</remarks>
+        <exception cref="T:System.InvalidOperationException">The current type is not a function pointer. That is, the <see cref="P:System.Type.`" /> property returns <see langword="false" />.</exception>
```

Comment:
The property is incomplete. Did you mean P:System.Type.IsFunctionPointer like in the exception below?

Reply:
Yes thanks